.github/workflows/ci.yml

```diff
@@ -443,6 +412,97 @@ ${{ hashFiles('requirements/requirements-python${{matrix.python-version}}.txt')
       - name: "Push CI image ${{ matrix.python-version }}"
         run: ./scripts/ci/images/ci_push_ci_image.sh
 
+  constraints:
+    timeout-minutes: 80
+    name: "Constraints"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+      fail-fast: false
+    needs:
+      - cancel-previous-workflow-run
+      - tests-sqlite
+      - tests-mysql
+      - tests-postgres
+      - tests-kubernetes
+    env:
+      PYTHON_MAJOR_MINOR_VERSION: ${{ matrix.python-version }}
+      CI_JOB_TYPE: "Requirements"
```

Comment:
Does this need to change?  (I am not sure --- just asking :) )

Reply:
aaaargh.. Missed that one :). In fact we do not need CI_JOB_TYPE at all any more (it was used in Travis). So I just deleted all of them now :)